Sources/Amplitude/AMPStorage.m

```diff
@@ -131,6 +129,7 @@ + (NSDictionary *)JSONFromFile:(NSString *)path {
     NSDictionary *jsonAsDict = [NSJSONSerialization JSONObjectWithData:data options:kNilOptions error:&err];
     if (err != NULL) {
         [AMPStorage finish:path];
+        data = [NSData dataWithContentsOfFile:path];
```

Comment:
I believe earlier pr also touched this method, let's pull the latest and make sure we don't have conflict and the new change won't break any tests.